### PR TITLE
Org info

### PR DESCRIPTION
**Pull request by benhylau**
https://github.com/hyphacoop/organizing/issues/190

### CODE REVIEW

File: co-operative.md [hunk 1]
```diff
@@ -8,15 +8,15 @@ Hypha Worker Co-operative Inc. is a worker co-operative incorporated on August
...
-and [145.1] of the Act.
+and role of members, as addressed in Article [1], [144.1], and [145] of the Act.
 
 We are a non-profit _organization_. 
 That means our organization does not operate with the primary purpose of generating profit. 
 As per 2017's [_Cutting Unnecessary Red Tape Act_](https://www.ontario.ca/laws/statute/s17020), 
 Hypha falls under neither the current [_Corporations Act_](https://www.ontario.ca/laws/statute/90c38), 
 nor the upcoming [_Not-for-Profit Corporations Act_](https://www.ontario.ca/laws/statute/10n15), 
 which both govern not-for-profit _corporations_ in Ontario.
+We are a [not-for-profit _co-operative_][not-for-profit-coop].
```
Comment (dcwalk) [inline co-operative.md:19]:
```suggestion
Instead, we are a [not-for-profit _co-operative_][not-for-profit-coop].
```

File: co-operative.md [hunk 2]
```diff
@@ -30,48 +30,59 @@ You can contact us the following ways:
...
-    - *active* if they are in good standing with their membership fees and have 
-      not requested to change their status
+    - *active* if they have not requested to change their status
     - *inactive* if they have requested to change their status
       - Inactive members are unable to vote, serve as a director, participate in 
         projects, or work as an employee for so long as they remain so
       - Inactive members can request to become active at any time
 
 An **employee**: 
+  - Works under a *contract of service* with Hypha
```
Comment (dcwalk) [inline co-operative.md:50]: can this be linked? I'm used to this term referring to self-employed folks

File: co-operative.md [hunk 3]
```diff
@@ -30,48 +30,59 @@ You can contact us the following ways:
...
 
 A **collaborator**:
   - Participates in open Hypha activities and projects without a formal employment
     relationship
 
-As a worker co-operative, there are specific conditions regarding employees and members, 
-addressed in Article [1], [1.1], [144] and [145.1] of the [*Ontario Co-operative Corporations Act*][coop-act]. 
+As every member holds a dual position as both a member and an employee of Hypha,
+they must make a distinction between *member duties* and *employee work*.
+Members are expected to acknowledge these components of their relationship with Hypha by signing the *Membership Acknowledgement*.
```
Comment (dcwalk) [inline co-operative.md:68]: We need to firm this thing up maybe before we add this line?

File: co-operative.md (hunk 2 above)
Comment (benhylau) [inline co-operative.md:50]: >It is clear that each organization calling itself a worker co-op must be assessed independently to determine whether there is a contract of service, i.e., employer/ employee relationship, or a contract for services, i.e., an independent contractor situation.

https://canadianworker.coop/guidelines-for-determining-whether-or-not-there-is-an-employer-employee-relationship-in-a-worker-co-op/

File: co-operative.md [hunk 4]
```diff
@@ -30,48 +30,59 @@ You can contact us the following ways:
...
+
+Employees, as well as contractors and collaborators engaged in certain initiatives with Hypha,
+are expected to sign an *Employee and Contractor Intellectual Property Agreement* and comply with
+[tax obligations as applicable to them](https://handbook.hypha.coop/finance.html#gsthst-rt).
+Hypha currently does not employ anyone as *member-contractor* or *dependent contractor*.
```
Comment (dcwalk) [inline co-operative.md:73]: what is a "member-contractor"? (If this is saying someone cant be a contractor + a member maybe we just say like that?)

File: co-operative.md (hunk 2 above)
Comment (dcwalk) [reply co-operative.md:50]:
```suggestion
  - Works as an *employee* under a *contract of service* with Hypha
```

File: co-operative.md (hunk 4 above)
Comment (dcwalk) [reply co-operative.md:73]:
```suggestion
```

File: co-operative.md [hunk 5]
```diff
@@ -30,48 +30,59 @@ You can contact us the following ways:
...
+  - May be a natural person or incorporated entity, domestic or foreign
 
 A **collaborator**:
   - Participates in open Hypha activities and projects without a formal employment
     relationship
 
-As a worker co-operative, there are specific conditions regarding employees and members, 
-addressed in Article [1], [1.1], [144] and [145.1] of the [*Ontario Co-operative Corporations Act*][coop-act]. 
+As every member holds a dual position as both a member and an employee of Hypha,
+they must make a distinction between *member duties* and *employee work*.
```
Comment (dcwalk) [inline co-operative.md:67]:
```suggestion
they must make a distinction between *member duties* and *employee work*.
Our bylaws do not allow for members to be contractors and we currently do not employ *dependent contractors*.
```

File: co-operative.md [hunk 6]
```diff
@@ -30,48 +30,59 @@ You can contact us the following ways:
...
   - Provides services clearly specified in a time-limited contractor agreement
+  - May be a natural person or incorporated entity, domestic or foreign
 
 A **collaborator**:
   - Participates in open Hypha activities and projects without a formal employment
     relationship
 
-As a worker co-operative, there are specific conditions regarding employees and members, 
-addressed in Article [1], [1.1], [144] and [145.1] of the [*Ontario Co-operative Corporations Act*][coop-act]. 
+As every member holds a dual position as both a member and an employee of Hypha,
```
Comment (dcwalk) [inline co-operative.md:66]:
```suggestion
As every member holds a dual position as both a member and an employee of Hypha,
```

File: co-operative.md (hunk 5 above)
Comment (dcwalk) [reply co-operative.md:67]: Moved up from below to here?

File: co-operative.md [hunk 7]
```diff
@@ -84,3 +95,4 @@ along with places we have a presence as a co-operative on our Loomio.</sup>
 [coop-act]: https://www.ontario.ca/laws/statute/90c35
 [es-act]: https://www.ontario.ca/laws/statute/00e41
 [esa-employee]: https://www.ontario.ca/document/changing-workplaces-review-final-report/chapter-8-who-employer-and-who-employee-under-employment-standards-act-2000#section-1
+[not-for-profit-coop]: https://ontario.coop/sites/default/files/STR06_For%20Profit%20and%20Not%20for%20Profit%20Co-ops.pdf
```
Comment (dcwalk) [inline co-operative.md:98]:
```suggestion
[not-for-profit-coop]: https://ontario.coop/sites/default/files/STR06_For%20Profit%20and%20Not%20for%20Profit%20Co-ops.pdf
[cwcf-guidelines]: https://canadianworker.coop/guidelines-for-determining-whether-or-not-there-is-an-employer-employee-relationship-in-a-worker-co-op/
```

**Comment by benhylau** [comment]
>do we need to have the membership acknowledgement drafted? Or remove that line until it is?

I think we can merge as is since that's tracked as a TODO.

**Pull request merged**